packages/react-components/src/molecules/ReminderItem.tsx

```diff
@@ -39,31 +39,75 @@ const descriptionStyles = css({
   },
 });
 
+const subtextStyles = css({
+  display: 'flex',
+  alignItems: 'start',
+  marginTop: `${12 / perRem}em`,
+  gap: `${8 / perRem}em`,
+});
+
+const timeElapsedStyles = css({
+  color: cerulean.rgb,
+});
+
 const iconMap: Record<ReminderEntity, React.ReactElement> = {
   Output: <LibraryIcon />,
   'Output Version': <LibraryIcon />,
   'Research Output': <LibraryIcon />,
   'Research Output Version': <LibraryIcon />,
   Event: <EventIcon />,
+  Manuscript: <LibraryIcon />,
 };
 
-type ReminderProps = Pick<ReminderResponse, 'description' | 'href'> & {
+type ReminderProps = Pick<
+  ReminderResponse,
+  'description' | 'href' | 'date' | 'subtext'
+> & {
   entity?: ReminderEntity;
 };
 
+export const getTimeElapsed = (isoDate: string): string => {
```

Comment:
the functions does the job, but there's a bit of redundancy in there, if I'm not being too picky 
```suggestion
export const getTimeElapsed = (isoDate: string): string => {
  const date = new Date(isoDate);
  const now = new Date();

  const secondsElapsed = Math.floor((now.getTime() - date.getTime()) / 1000);

  if (secondsElapsed < 3600) {
    return `${Math.floor(secondsElapsed / 60)}m`;
  }
  if (secondsElapsed < 86400) {
    return `${Math.floor(secondsElapsed / 3600)}h`;
  }
  return `${Math.floor(secondsElapsed / 86400)}d`;
};

```

Reply:
Thanks! I've updated according to your suggestion :)